index.html

```diff
@@ -9,11 +9,11 @@
   <body>
     <div>
       <header>
-        <h2 class="studentName">Student Page</h2>
+        <h2 class="Bashaer">Bashaer's Page</h2>
       </header>
 
       <section class="container">
-        <h3 class="aboutMe">About me</h3>
+        <h3 class="aboutMe"> A palestinian student ,I love learning new things, my passions are writing, singing and drawing </h3>
```

Comment:
Awesome :relaxed: 
just update this section so your added description will be inside `<p>` tag not an `<h3>` tag, like so:
```html
        <h3 class="aboutMe">About me</h3>
        <p>Add your description here </p>
```